zerobuf/Zerobuf.h

```diff
@@ -47,7 +47,7 @@ class Zerobuf
     ZEROBUF_API Zerobuf& operator = ( const Zerobuf& rhs );
     Allocator* getAllocator() { return _alloc; }
 
-    void _setZerobufArray( const void* data, const size_t size,
+	ZEROBUF_API void _setZerobufArray(const void* data, const size_t size,
```

Comment:
You guys are really counting spaces, right? Unfortunately I'll be out of office from now through the next week. So if anyone could merge and fix this, I'd be grateful. Promise to be more thorough next time around...